shared/profile/routes.js

```diff
@@ -58,4 +51,8 @@ const routeTree = new RouteDefNode({
   },
 })
 
+const routeTree = new RouteDefNode({
+  ...profileRoute.toJS(),
+})
+
 export default routeTree
```

Comment:
I think you can just export `profileRoute` here, since you're not modifying it in any way.